docs/python/testing.md

```diff
@@ -234,6 +234,21 @@ Support for running tests in parallel with pytest is available through the `pyte
 
 3. Run your tests, which will now be run in parallel.
 
+## Run tests with coverage
+
+Test coverage is a measure of how much of your code is covered by your tests, which can help you identify areas of your code that are not being fully tested. For more information on test coverage, visit VS Code's [Test Coverage documentation](/docs/editor/testing#test-coverage.md).
```

Comment:
```suggestion
Test coverage is a measure of how much of your code is covered by your tests, which can help you identify areas of your code that are not being fully tested. For more information on test coverage, visit VS Code's [Test Coverage documentation](/docs/editor/testing#_test-coverage.md).
```

Reply:
I'm not sure this is right but the link was broken to me. Going to the [URL ](https://code.visualstudio.com/docs/editor/testing#_test-coverage)it seemed it needed a `_` but it'd be good to double check